README.md

```diff
@@ -1,286 +1,116 @@
 # React components for TalkJS
 
-This library makes it easy to use the [TalkJS](https://talkjs.com) pre-built chat UIs inside a React web application.
+This library makes it easy to use [TalkJS](https://talkjs.com) pre-built chat UIs inside a React web application.
 
-`@talkjs/react` encapsulates `talkjs`, the framework-independent ("vanilla") [TalkJS JavaScript SDK](https://talkjs.com/docs/Reference/JavaScript_Chat_SDK). It only provides React components for UI-related matters: For anything related to data manipulation, such as synchronizing user data, or creating and joining conversations, use the vanilla SDK.
+`@talkjs/react` encapsulates `talkjs`, the framework-independent [TalkJS JavaScript SDK](https://talkjs.com/docs/Reference/JavaScript_Chat_SDK). It provides React components for UI-related matters. For anything related to data manipulation, such as synchronizing user data, or creating and joining conversations, use the regular SDK.
 
 TypeScript bindings are included.
 
 If you encounter any problems with `@talkjs/react`, please open an issue. If you have a problem with TalkJS itself, or if you're not sure where the problem lies, it's better to open a chat with our support on https://talkjs.com/ (just open the chat widget). TalkJS support is staffed by engineers.
```

Comment:
```suggestion
If you encounter any problems with `@talkjs/react`, please open an issue. If you have a problem with TalkJS itself, or if you're not sure where the problem lies, it's better to open a [chat for support](https://talkjs.com/?chat). TalkJS support is staffed by engineers.
```
I believe if you use this link then it automatically opens the chat window :)

Reply:
Nice, TIL!